clients/src/main/java/org/apache/kafka/clients/consumer/CommitFailedException.java

```diff
@@ -28,7 +28,12 @@
 
     private static final long serialVersionUID = 1L;
 
-    public CommitFailedException(String message) {
-        super(message);
+    public CommitFailedException() {
+        super("Commit cannot be completed since the group has already " +
+                "rebalanced and assigned the partitions to another member. This means that the time " +
```

Comment:
Commits are accepted while the rebalance in progress. It's not until it finishes that the generation is incremented and we start rejecting commits.